src/app.js

```diff
@@ -29,8 +30,18 @@
   _.extend(spiderOakApp, {
     config: window.spiderOakMobile_config,     // Supplemented in initialize.
     ready: function() {
+      var soApp = window.spiderOakApp,
```

Comment:
Just to be shorter. Not crucial, could be removed.

(I had originally included a commented-out line for revealing functionality, and needed to shorten it so it only took one line. )